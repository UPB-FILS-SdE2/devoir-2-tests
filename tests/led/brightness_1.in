led brightness set 2 0 0
led brightness 0 0
led on 0 0
led brightness 0 0
quit