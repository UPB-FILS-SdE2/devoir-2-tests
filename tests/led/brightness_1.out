cmd: Set pixel 0 0 to 2
cmd: 2
cmd: Set pixel 0 0 to 9
cmd: 9
cmd: 